docs/dev-env.md

```diff
@@ -12,7 +12,12 @@ and follow the instructions instead of reading this document.
 
 ## Clone repository
 
-Clone this repository locally, with `git clone <repo-url>`, then `cd <repo-dir>`,
+Clone the fedimint git repository locally and cd into it:
+
+```
+git clone git@github.com:fedimint/fedimint.git
+cd fedimint
+```
```

Comment:
Easier to copy-paste. If user wants to clone over https they can figure make the change themselves.